src/Config_Command.php

```diff
@@ -224,6 +230,14 @@ public function get( $_, $assoc_args ) {
 		$wp_config_vars      = self::get_wp_config_vars( get_defined_vars(), $wp_cli_original_defined_vars, 'variable', array( 'wp_cli_original_defined_vars' ) );
 		$wp_config_constants = self::get_wp_config_vars( get_defined_constants(), $wp_cli_original_defined_constants, 'constant' );
 
+		$get_constant        = ! empty( $assoc_args['constant'] );
+		$get_global          = ! empty( $assoc_args['global'] );
```

Comment:
If both `$get_constant` _and_ `$get_global` are provided, we should return an error. This error condition should also have a test.